flink-ai-flow/ai_flow/plugin_interface/blob_manager_interface.py

```diff
@@ -18,9 +18,40 @@
 #
 from abc import ABC, abstractmethod
 from typing import Text, Dict
+
+from ai_flow.common.configuration import AIFlowConfiguration
 from ai_flow.common.module_load import import_string
 
 
+class BlobConfig(AIFlowConfiguration):
```

Comment:
Now that the `BlobConfig` and `BlobManagerFactory` are defined in `blob_manager_interface.py`, we'd better move `SchedulerFactory` and `SchedulerServiceConfig` to `scheduler_interface.py`

Reply:
I create an issue for this. And I think it is better to start this refractor in a new pr due to such changes is relevant to a lot of files and needs to be more careful.